beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactory.java

```diff
@@ -135,6 +138,14 @@ public Consumer<BeaconBlockBodyBuilder> createSelector(
 
       final SpecVersion specVersion = spec.atSlot(blockSlotState.getSlot());
 
+      if (specVersion.getMilestone().isGreaterThanOrEqualTo(SpecMilestone.CAPELLA)) {
+        final SszList<SignedBlsToExecutionChange> blsToExecutionChanges =
+            BeaconBlockBodySchemaCapella.required(
+                    specVersion.getSchemaDefinitions().getBeaconBlockBodySchema())
+                .getBlsToExecutionChangesSchema()
+                .getDefault();
+        bodyBuilder.blsToExecutionChanges(() -> blsToExecutionChanges);
+      }
```

Comment:
We shouldn't need the if statement here - the function passed to `bodyBuilder.blsToExecutionChanges` should only be called if Capella is active, so you can just do:
```suggestion
        bodyBuilder.blsToExecutionChanges(() -> {
            return BeaconBlockBodySchemaCapella.required(
                    specVersion.getSchemaDefinitions().getBeaconBlockBodySchema())
                .getBlsToExecutionChangesSchema()
                .getDefault();
        });
```

(As usual this is guaranteed to fail spotless...)

Reply:
yes wasn't going to attempt to accept that change :)